File: Iniciante/1012.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args){
 
        Scanner scanf = new Scanner(System.in);

        float triangulo, circulo, trapezio, quadrado, retangulo;
        float a = scanf.nextFloat();
        float b = scanf.nextFloat();
        float c = scanf.nextFloat();

    triangulo = (float) (a * c / 2.0);
    circulo = (float) (3.14159 * (c * c));
    trapezio = (float) (((a + b) * c) / 2.0);
    quadrado = (float) b * b;
    retangulo = (float) a * b;

    System.out.printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n", triangulo, circulo, trapezio, quadrado, retangulo);
 
    }
 
}

File: Iniciante/1012.py
a, b, c = input().split(" ")
a = float(a)
b = float(b)
c = float(c)

triangulo = a * c / 2.0
circulo = 3.14159 * (c * c)
trapezio = ((a + b) * c) / 2.0
quadrado = b * b
retangulo = a * b

print(f'TRIANGULO: {triangulo:.3f}')
print(f'CIRCULO: {circulo:.3f}')
print(f'TRAPEZIO: {trapezio:.3f}')
print(f'QUADRADO: {quadrado:.3f}')
print(f'RETANGULO: {retangulo:.3f}')